src/main/java/com/google/api/codegen/viewmodel/StaticXApiView.java

```diff
@@ -0,0 +1,87 @@
+/* Copyright 2016 Google Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.api.codegen.viewmodel;
+
+import com.google.api.codegen.SnippetSetRunner;
+import com.google.auto.value.AutoValue;
+
+import java.util.List;
+
+@AutoValue
+public abstract class StaticXApiView implements ViewModel {
```

Comment:
OK yes that makes sense. When reading the name, I thought of static as in static methods. Do you think it would be an improvement to rename to StaticLanguageXApiView? (or StaticLangXApiView?) Or would this just be more verbose without additional clarity...